beacon-chain/rpc/beaconv1/blocks_test.go

```diff
@@ -0,0 +1,596 @@
+package beaconv1
+
+import (
+	"bytes"
+	"context"
+	"reflect"
+	"testing"
+
+	mockPOW "github.com/prysmaticlabs/prysm/beacon-chain/powchain/testing"
+
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1"
+	ethpb_alpha "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	mock "github.com/prysmaticlabs/prysm/beacon-chain/blockchain/testing"
+	"github.com/prysmaticlabs/prysm/beacon-chain/db"
+	dbTest "github.com/prysmaticlabs/prysm/beacon-chain/db/testing"
+	mockp2p "github.com/prysmaticlabs/prysm/beacon-chain/p2p/testing"
+	p2ppb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/proto/migration"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/shared/params"
+	"github.com/prysmaticlabs/prysm/shared/testutil"
+	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
+	"github.com/prysmaticlabs/prysm/shared/testutil/require"
+)
+
+func fillDBTestBlocks(ctx context.Context, t *testing.T, db db.Database) (*ethpb_alpha.SignedBeaconBlock, []*ethpb_alpha.BeaconBlockContainer) {
+	// Should return the proper genesis block if it exists.
```

Comment:
What does this mean?

Reply:
Might've been from a copy/paste, removed.